imagery/i.ortho.photo/lib/find_init.c

```diff
@@ -8,11 +8,15 @@
 int I_find_initial(char *group)
 {
     char *element;
-
-    element = (char *)G_malloc(80 * sizeof(char));
+    int file_exists;
 
     if (group == NULL || *group == 0)
         return 0;
-    sprintf(element, "group/%s", group);
-    return G_find_file(element, "INIT_EXP", G_mapset()) != NULL;
+
+    element = (char *)G_malloc(GNAME_MAX * sizeof(char));
```

Comment:
I think it would be better to declare element as:

```c
char element[GNAME_MAX];
```

then no need to malloc, free...

Reply:
As `group` may be of size `GNAME_MAX`

https://github.com/OSGeo/grass/blob/20985774a0965549fcc43c500f697aac5172c546/imagery/i.ortho.photo/lib/orthophoto.h#L93-L94

snprintf may truncate, so you should check for that too.

Reply:
Changes done.
for snprintf check i have used G_warning and return 0. Is it okay or should i just use G_fatal and not include return 0